test/groovy/com/sap/piper/variablesubstitution/YamlUtilsTest.groovy

```diff
@@ -0,0 +1,276 @@
+package com.sap.piper.variablesubstitution
+
+import org.junit.Before
+import util.JenkinsDeleteFileRule
+
+import static org.junit.Assert.*
```

Comment:
Strange syntax: normally there is no `.*` after the class.

Reply:
It's _statically_ importing. I.e. it imports all static functions of `Assert`, which is a very common syntax actually.

Reply:
you are right, I overlooked the `static`  ...